workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -42,17 +40,18 @@ protected override void OnUpdate()
 
         private void OnAddEntity(AddEntityOp op)
         {
-            view.CreateEntity(op.EntityId.Id);
+            worker.CreateEntity(op.EntityId.Id);
```

Comment:
I wonder if its worth inlining what the worker does here? It looks like that method was moved from the View to the Worker

Reply:
maybe. I would prefer to keep it in the worker for now as we will need to identify how to expose the entity mapping. but I'll keep it in mind for the next PR and probably move it over there